tools/df/docker/bird-build/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG BASE_IMAGE=276327358223.dkr.ecr.us-east-1.amazonaws.com/pipedream-dev
```

Comment:
pipedream-dev has essentially our whole product in it, are there deepfield-specific dependencies we need to build bird?